# This is a message that holds state of wheeled bipedal robot Pow.
#
# The state of wheeled bipedal robot Pow is
#  * the pitch angle
#  * the pitch angular rate
#  * the velocity of x-axis
#  * the yaw angular rate

float64 theta
float64 theta_dot
float64 vel
float64 psi_dot